docs/quick-start/chat-agent-tool.md

```diff
@@ -66,7 +66,7 @@ Essentially the `ProbeTool` definition specifies
 
 Here is what the `ProbeTool` definition looks like:
 ```py
-class ProbeTool(ToolMessage):
+class ProbeTool(lr.agent.ToolMessage):
```

Comment:
`ToolMessage` is worth exposing at top level

Reply:
actually never mind let's not expose `ToolMessage`. We want it to be consistent with the specific tools like `RecipientTool` etc... exposing `ToolMessage` will make usage inconsistent.